### ISSUE TITLE
Change allow_stopping_for_update from true to false

### ISSUE THREAD

**Issue by benbro** [terraform-providers/terraform-provider-google#1186]
It should be possible to change the "allow_stopping_for_update" field without restarting the server if all other fields like machine type aren't changed.

### Terraform Version
Terraform v0.11.3
+ provider.google v1.6.0
+ provider.local v1.1.0

### Affected Resource(s)
- google_compute_instance

### Terraform Configuration Files
```hcl
resource "google_compute_instance" "mytest" {
  name         = "mytest"
  machine_type = "n1-standard-1"
  zone         = "europe-west1-b"
  allow_stopping_for_update = true

  scheduling {
    automatic_restart = true
  }

  tags = ["mytag"]

  boot_disk {
    initialize_params {
      size = 20
      image = "ubuntu-os-cloud/ubuntu-1604-lts"
    }
  }

  network_interface {
    network = "default"

    access_config {
      // Ephemeral IP
    }
  }

  service_account {
    email = "${google_service_account.mytest.email}",
    scopes = ["https://www.googleapis.com/auth/cloud-platform"]
  }
}
```

### Expected Behavior
The field should change without restarting the server:
```
Terraform will perform the following actions:
  ~ google_compute_instance.mytest
      allow_stopping_for_update: "true" => "false"
```

### Actual Behavior
```
Error: Error applying plan:

1 error(s) occurred:

* google_compute_instance.mytest: 1 error(s) occurred:

* google_compute_instance.mytest: Changing the machine_type, min_cpu_platform, or service_account on an instance requires stopping it. To acknowledge this, please set allow_stopping_for_update = true in your config.
```

### Steps to Reproduce
1. create compute instance without the allow_stopping_for_update field.
2. change the machine type and set "allow_stopping_for_update = true" .
3. change "allow_stopping_for_update = false"

**Comment by danawillow** [comment]
Hey @benbro, I think this was caused by the issue that #1130 fixed. We just released that fix today- can you try upgrading to 1.7.0 and seeing if that helps?

**Comment by benbro** [comment]
It is fixed in 1.7.0.
Thank you for the fast response.
